### ISSUE TITLE
Inserting a pause into pptx by `. . .` does not work

### ISSUE THREAD

**Issue by ciaochiaociao** [jgm/pandoc#5701]
```
# Slide with a pause

content before the pause

. . .

content after the pause
```

Just tried the example given in your documentation and it does not work.

Note: I am using PowerPoint 2017 now.

**Comment by mb21** [comment]
Most of the text under "Producing slide shows with pandoc" in the MANUAL was written for html and latex slide-shows. We should indeed clarify which sections don't work with the powerpoint writer (yet).

**Comment by jkr** [comment]
Yeah -- this is a subset of animations not being implemented. I have to admit the animation ooxml language has been too much for me so far (even just for writing a subset). It's really the one big thing left for this format. Someday...

In the meantime, this is a documentation issue. I'd be happy to do it soon, but anyone is welcome if they get to it first.
